src/core/configuration/DevConfig.ts

```diff
@@ -42,6 +42,10 @@ export class DevServerConfig extends DefaultServerConfig {
   subdomain(): string {
     return "";
   }
+
+  replayFallbackUrl(gameId: GameID): string {
+    return `https://api.openfront.io/game/${gameId}`;
```

Comment:
We should be consistent about either using or not using the URL class to urlencode the path, so that if there's a bug in either approach, we're able to reproduce the behavior we see in prod in our local builds.

Using URL is sensible if the `gameId` could contain un-sanitized user input, for example if we didn't use zod to verify that it was a valid `ID`.

```suggestion
    const url = new URL("https://api.openfront.io");
    url.pathname = `/game/${gameId}`;
    return url.toString();
```